searx/engines/command.py

```diff
@@ -0,0 +1,184 @@
+'''
+searx is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+searx is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with searx. If not, see < http://www.gnu.org/licenses/ >.
+'''
+
```

Comment:
Just some minor nit-picking, the licences text can be replaced by (https://github.com/searx/searx/issues/1801):

    # SPDX-License-Identifier: AGPL-3.0-or-later

Reply:
Yes ! there are a lot of other files to modified too, perhaps it is better to address this issue in another PR (even

Reply:
This file is new created and I thought it is more straight forward to tag new files from the beginning.